### PR TITLE
Freemius SDK Upgrade - WP feedback incorporate

### PR DESCRIPTION
**Pull request by abuzar347**

### CODE REVIEW

**Review by afifa-glowlogix — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file, per [the Ke
 
 ## [Unreleased]
```
[the remark this answers is not in the document]
Comment (musamamasood) [reply CHANGELOG.md]: Still there and linked to the master branch and the latest release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file, per [the Ke
...
 
+## [1.2.6] - 2022-02-12
```
Comment (afifa-glowlogix) [inline CHANGELOG.md:7]: @abuzar347 release date for 1.2.6 is 2022-04-22
Also, match the format of the last two releases to the order releases and update fixes as well.

**Review by afifa-glowlogix — CHANGES_REQUESTED** [review]

**Comment by afifa-glowlogix** [comment]
@abuzar347  release date for 1.2.6 version is 04/22/2022, not 5th April.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,10 +4,10 @@ All notable changes to this project will be documented in this file, per [the Ke
 
 ## [Unreleased]
 
-## [1.2.6] - 2022-02-12
+## [1.2.6] - 2022-04-05
```
Comment (afifa-glowlogix) [inline CHANGELOG.md:7]: @abuzar347  release date for 1.2.6 version is 04/22/2022, not 5th April.

**Review by afifa-glowlogix — CHANGES_REQUESTED** [review]

**Review by afifa-glowlogix — APPROVED** [review]

**Pull request merged**